src/windows/process.rs

```diff
@@ -547,6 +547,18 @@ impl ProcessExt for Process {
     fn group_id(&self) -> Option<Gid> {
         None
     }
+
+    fn wait(&self) -> Option<Pid> {
+        let (mut status, mut _pid): (libc::c_int, i32) = (0, -1);
+        let _ppid =  self.parent().unwrap().as_u32();
+        let _uppid = std::process::id();
+        if _ppid == _uppid {
+            let process_handler = get_process_handler(self.pid.0);
+            winapi::um::synchapi::WaitForSingleObject(process_handler, -1);
```

Comment:
So I'm not sure if the `HANDLE` has the correct right to be able to be waited on.

This code would do it I suppose:

```rust
if let Some(handle) = proc_.get_handle() {
    while is_proc_running(handle) {
        if get_start_time(handle) != proc_.start_time() {
            // PID owner changed so the previous process was finished!
            return
        }
        std::thread::sleep(std::time::Duration::from_millis(10));
    }
} else {
    // In this case, we can't do anything so we just return.
    sysinfo_debug!("can't wait on this process so returning");
}
```